src/Apps/Artwork/Components/ArtworkBanner/index.tsx

```diff
@@ -0,0 +1,207 @@
+import { renderWithLoadProgress } from "Artsy/Relay/renderWithLoadProgress"
+import { ContextConsumer } from "Artsy/Router"
+import React from "react"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import { get } from "Utils/get"
+import { Banner } from "./Banner"
+
+import { ArtworkBanner_artwork } from "__generated__/ArtworkBanner_artwork.graphql"
+import { ArtworkBannerQuery } from "__generated__/ArtworkBannerQuery.graphql"
+
+export interface ArtworkBannerProps {
+  artwork: ArtworkBanner_artwork
+}
+
+export const ArtworkBanner: React.SFC<ArtworkBannerProps> = props => {
+  const {
+    artworkContextAuction,
+    artworkContextFair,
+    artworkContextPartnerShow,
+    partner,
+  } = props.artwork
+
+  // imageUrl: image for avatar
+  // initials: fallback partner initials in case image is not there.
+  // meta: in auction / at fair / in show
+  // name:  auction / fair / show name
+  // subHeadline: partner name
+  if (artworkContextAuction) {
```

Comment:
I think this series of `if`s here with the corresponding shortcut returns actually read clear here and make it more explicit that only one banner will be displayed at a time. 

But if we do decide to go with proper query without aliasing once it's a possibility - the change is easy here.